src/py/flwr/client/app.py

```diff
@@ -373,7 +373,7 @@ def _on_backoff(retry_state: RetryState) -> None:
             )
 
     retry_invoker = RetryInvoker(
-        wait_gen_factory=exponential,
+        wait_gen_factory=lambda: exponential(max_delay=20),
```

Comment:
@panh99 should we move the `20` to `constants`?

Reply:
Sure.